Makefile

```diff
@@ -15,52 +15,56 @@ clean: rwundo clean_sharness
 	@rm -rf ./test/testingData
 	@rm -rf ./compose
 
-install: deps
+install: gx-deps
 	$(MAKE) -C cmd/ipfs-cluster-service install
 	$(MAKE) -C cmd/ipfs-cluster-ctl install
 
-docker_install: docker_deps
+docker_install: docker_gx-deps
 	$(MAKE) -C cmd/ipfs-cluster-service install
 	$(MAKE) -C cmd/ipfs-cluster-ctl install
 
-build: deps
+build: gx-deps
 	go build -ldflags "-X ipfscluster.Commit=$(shell git rev-parse HEAD)"
 	$(MAKE) -C cmd/ipfs-cluster-service build
 	$(MAKE) -C cmd/ipfs-cluster-ctl build
 
-service: deps
+service: gx-deps
 	$(MAKE) -C cmd/ipfs-cluster-service ipfs-cluster-service
-ctl: deps
+ctl: gx-deps
 	$(MAKE) -C cmd/ipfs-cluster-ctl ipfs-cluster-ctl
 
 gx-clean: clean
 	$(MAKE) -C $(deptools) gx-clean
 
 gx:
+ifeq ($(GX),yes)
```

Comment:
A comment somewhere saying the env var `GX` needs to be set to `yes` for the gx make targets to work.